extensions/stackdriver/testdata/stackdriver_filter.yaml

```diff
@@ -21,7 +21,7 @@ spec:
             config:
               root_id: stackdriver_outbound
               configuration: |
-                {}
+                {"enable_mesh_edges_reporting": false, "meshEdgesReportingDuration": "600s"}
```

Comment:
The default is 600s. No need to explicitly config it.

Reply:
That way people know which arg to change if needed.
Same reason why we have enable_mesh_edges: false

Reply:
Ah, I see.